src/db/migrations/20231220015816_onerep_stats.js

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export async function up(knex) {
+  return knex.schema
+    .createTable('onerep_stats', table => {
```

Comment:
Should this be limited to OneRep only or do we want to open this up for other stats in the future? 

ie. We can distinguish with a `type` col